src/Lumi/Components/Form.purs

```diff
@@ -764,6 +768,59 @@ arrayModal { label, addLabel, defaultValue, summary, component, componentProps }
       , validate: pure xs
       }
 
+editableTable
+  :: forall props row result
+   . { addLabel :: String
+     , defaultValue :: row
+     , form :: FormBuilder { readonly :: Boolean | props } row result
+     , labels :: Array String
+     , maxRows :: Int
+     , summary :: JSX
+     }
+  -> FormBuilder
+      { readonly :: Boolean
+      | props
+      }
+      (Array row)
+      (Array result)
+editableTable { addLabel, defaultValue, form, labels, maxRows, summary } =
+  FormBuilder \props@{ readonly } rows ->
+    let
+      fs = (un FormBuilder form) props <$> rows
+    in
+      { edit: \onChange -> pure $
+          Child
+            { key: Nothing
+            , child:
+                EditableTable.editableTable
+                  { addLabel
+                  , maxRows
+                  , readonly
+                  , rows: Left (Array.mapWithIndex Tuple rows)
+                  , rowEq: unsafeRefEq
```

Comment:
Would ``eq `on` fst`` work here?

Reply:
I just looked at the EditableTable component and `rowEq` isn't used any more actually, so we should probably think about removing that prop.